.rubocop_todo.yml

```diff
@@ -10,6 +10,41 @@
 InternalAffairs/NodeDestructuring:
   Enabled: false
 
+# It is not intended that the following exclusions be merged. These exclusions
+# are temporary pending a maintainer decision re: the current discrepancy
+# between eg:
+#
+#     x = begin
+#       foo
+#     end
+#
+# and
+#
+#     x = if y
+#           foo
+#         end
+#
+# EndAlignment recently began to care about the former (`begin`) thus making
+# these temporary exclusions necessary.
```

Comment:
What I'm trying to say here is that rubocop itself uses the `keyword` style for eg. `if`, but uses the `start_of_line` style for `begin`.